src/locales/de/translation.json

```diff
@@ -197,6 +205,7 @@
   "Public access token for LocationIQ services.": "Öffentlicher Zugriffstoken für LocationIQ-Dienste.",
   "Style Renderer": "Stil-Renderer",
   "Choose the default Maputnik renderer for this style.": "Wähle den Standard-Renderer für diesen Stil aus.",
+  "Press <1>ESC</1> to lose focus": "Drück <1>ESC</1>, um den Fokus zu verlieren",
```

Comment:
Is this translation still correct? (I was looking into this as the german wording is akward)

https://github.com/maplibre/maputnik/blob/454d8d8b108b5a14c3a5186953b97ea2a0fee6d3/src/components/modals/ModalShortcuts.tsx#L113

Reply:
This one just moved, don't ask me why i18n doesn't sort the file by key instead...